Patches/Dishonored Assassin Coat/ThingDefs_Misc/Apparel.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Dishonored Assassin Coat</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+		    <operations>
+				<!-- recipeEdits -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Apparel_AssassinCoatMale"]/costStuffCount</xpath>
+					<value>
+						<costStuffCount>100</costStuffCount>
+					</value>
+				</li>
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Apparel_AssassinCoatMale"]</xpath>
+					<value>
+						<costList>
+							<Cloth>25</Cloth>
+						</costList>
+					</value>
+				</li>
+				<!-- statBases -->
+				<li Class="PatchOperationReplace">
+			        <xpath>Defs/ThingDef[defName="Apparel_AssassinCoatMale"]/statBases/StuffEffectMultiplierArmor</xpath>
+					<value>
+						<Bulk>1.5</Bulk>
+						<WornBulk>4</WornBulk>
+						<StuffEffectMultiplierArmor>6</StuffEffectMultiplierArmor>
+					</value>
+			    </li>
+				<!-- Misc -->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Apparel_AssassinCoatMale"]/apparel/layers</xpath>
+					<value>
+						<li>Webbing</li>

```

Comment:
Why are you using Webbing? I'd switch it to Shell

Reply:
I concur, I'd make it the same layer as the Duster.